src/default/assets/js/src/typedoc/components/Filter.ts

```diff
@@ -115,11 +109,19 @@ namespace typedoc
             $html.removeClass('toggle-' + oldValue);
             $html.addClass('toggle-' + newValue);
         }
+
+        protected fromLocalStorage(value: string): string {
+            return value;
```

Comment:
This was already like this but it is odd to me that this just passes the value through 🤔

Reply:
The filter stores a string in localStorage, so when it is retrieved, we don't need to do anything.